### PR TITLE
Added kafka-support related changes

### PR DESCRIPTION
**Pull request by vaibhavatul47**
- Updated docker-compose with kafka related ports.
- Added new sample for kafka produce-consume

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,30 @@ To get started, refer to our GitHub Samples [here](https://github.com/Azure/azur
   
 >[!TIP]
 > $Default [consumer group](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-features#consumer-groups) is created by default when emulator runs. You can't create $default consumer group with supplied configuration.
+
+### Networking options
+You can run and connect to Emulator in multiple ways. Use a `Connection String` from following as per your use-case:
+
+- When the emulator container and interacting application are running natively on local machine:
+```
+"Endpoint=sb://localhost;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;"
```
Comment (SayanMS09) [inline README.md:98]: Can we use this exact string (with the "UseDevelopmentEmulator=true") at the end in our .net sample code too?

**Review by SayanMS09 — CHANGES_REQUESTED** [review]
Use same connection string which is provided in README and in sample code.

**Comment by vaibhavatul47** [comment]
@microsoft-github-policy-service agree [company="microsoft"]

**Review by SayanMS09 — APPROVED** [review]
Approved. Lets wait for Sannidhya's Approval and right time to merge this change.

**Review by Saglodha — APPROVED** [review]

**Pull request merged**